action.yml

```diff
@@ -22,6 +22,9 @@ inputs:
     description: The domain of the Buf Schema Registry to login to.
     required: false
     default: 'buf.build'
+  go_version_file:
```

Comment:
Not sure about naming here.

Reply:
This matches the name that [action/setup-go](https://github.com/actions/setup-go) does with underscores instead of dashes to follow your existing naming convention.

Reply:
If you prefer just `version_file` that would work too?